modules/openapi-generator/src/main/resources/JavaJaxRS/resteasy/pojo.mustache

```diff
@@ -1,4 +1,5 @@
 import io.swagger.annotations.*;
+{{#useBeanValidation}}import javax.validation.Valid;{{/useBeanValidation}}
```

Comment:
FYI. You can do the following to avoid a blank line if `useBeanValidation` is not true
```
{{#useBeanValidation}}
import javax.validation.Valid;
{{/useBeanValidation}}
```